app/views/layouts/_header.html.erb

```diff
@@ -68,23 +68,23 @@
         <li><%= link_to 'Employer Services', employers_path %></li>
         <li><%= link_to 'Web Development', deploy_path %></li>
         <li role="separator" class="divider"></li>
-      
+
         <li><%= link_to 'Success Stories', success_stories_path %></li>
         <li><%= link_to 'Blog', 'https://medium.com/operation-code' %></li>
         <li role="separator" class="divider"></li>
-        
+
         <a class="btn" href="https://donorbox.org/operationcode"
             data-constrainwidth="false" data-beloworigin="true" data-activates="Donate">Donate</a>
-            
+
         <li role="separator" class="divider"></li>
 
         <% if veteran_signed_in? %>
           <li><%= link_to 'Profile', profile_path %></li>
           <li><%= link_to 'Sign Out', destroy_veteran_session_path, method: :delete %></li>
         <% else %>
           <li>
-            <% ab_test(:signup_button_size, 'btn btn-large', '') do |button_size| %>
-            <%= link_to 'Join', new_veteran_registration_path, class: button_size, style: 'background-color: #B23900;' %>
+            <% ab_test(:signup_button_size, 'btn btn-large', 'btn') do |button_size| %>
+            <%= link_to 'Join', new_veteran_registration_path, class: 'btn', style: 'background-color: #B23900;' %>
```

Comment:
When I change the class back to `button_size`, here's what happens:
![button_size-bug](https://cloud.githubusercontent.com/assets/8031689/23151446/2f587a8a-f7c0-11e6-861f-b96be857f57b.png)
You even mentioned that this was the issue [here](https://github.com/OperationCode/operationcode/issues/637#issuecomment-281117519).

I've changed both ab_tests to read:
```
<% ab_test(:signup_button_size, 'btn btn-large', 'btn') do |button_size| %>
<%= link_to 'Join', new_veteran_registration_path, class: 'button_size', style: 'background-color: #B23900;' %>
```
yet the bug still persists.